src/api-service/__app__/timer_tasks/__init__.py

```diff
@@ -19,7 +19,7 @@ def main(mytimer: func.TimerRequest, dashboard: func.Out[str]) -> None:  # noqa:
     expired_tasks = Task.search_expired()
     for task in expired_tasks:
         logging.info("stopping expired task: %s", task.job_id)
-        task.stopping()
+        task.mark_stopping()
```

Comment:
How does this change relate to the other? Is it required?

Reply:
It is required, such that instead of calling the state handler directly, it does all of the other related stuff.

State handling is something we could use another refactor on, but that's out of scope for fixing this issue.